src/ai/backend/gateway/stream.py

```diff
@@ -322,6 +322,8 @@
     t.Dict({
         tx.AliasedKey(['app', 'service']): t.String,
         tx.AliasedKey(['port'], default=None): t.Null | t.Int[1024:65535],
+        tx.AliasedKey(['arguments'], default=None): t.Null | t.String, # {'-P': '12345'}
+        tx.AliasedKey(['envs'], default=None): t.Null | t.String # {'PASSWORD': '12345'}
```

Comment:
Insert an additional space before the comment sign (`#`) as PEP-8 recommends to use *two or more* spaces before line-ending comments.